servers/rendering/storage/render_scene_data.cpp

```diff
@@ -29,24 +29,25 @@
 /**************************************************************************/
 
 #include "render_scene_data.h"
+#include "render_scene_data.compat.inc"
 
 void RenderSceneData::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_cam_transform"), &RenderSceneData::get_cam_transform);
-	ClassDB::bind_method(D_METHOD("get_cam_projection"), &RenderSceneData::get_cam_projection);
+	ClassDB::bind_method(D_METHOD("get_cam_projection", "flip_y"), &RenderSceneData::get_cam_projection);
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("get_cam_projection", "flip_y"), &RenderSceneData::get_cam_projection, DEFVAL(true));
```
To match the c++, also otherwise the compatibility is broken

Reply:
Ta, missed that one!